src/states/Menu.gd

```diff
@@ -6,6 +6,7 @@ func _ready() -> void:
 		$V/Buttons/QuitButton.hide()
 	$V/Buttons/PlayButton.grab_focus()
 	Global.current_level_idx = 0
+#	get_tree().get_root().set_transparent_background(true)		# Transparent background for screenshots
```

Comment:
This could also be part of the debug mode. We also had to edit the project.godot file to enable per-pixel transparency and change the clear color - my guess is we can probably do those things via code instead and we can do them in debug mode